megatron/model/transformer.py

```diff
@@ -144,6 +137,8 @@ def forward(self, hidden_states):
 
         # [s, b, h]
         output, output_bias = self.dense_4h_to_h(intermediate_parallel)
+        output, output_bias = gather(output), gather(output_bias)
```

Comment:
Gathers the divided outputs and output biases, such that they are now replicated across all tensor parallel ranks. Essentially the opposite of drop.